### ISSUE TITLE
Relying on setInterval

### ISSUE THREAD

**Issue by DavidRHulme** [slorenzo/react-stopwatch#6]
From a metronome app I wrote, I don't think you can rely on setInterval for timing if you want it to be precise. Over time it will drift. Use can use something like performance.now() instead.

**Comment by nermand** [comment]
Agree. It is a Google Audit suggested method, described here:
https://developers.google.com/web/tools/lighthouse/audits/date-now

**Comment by saeta-eth** [comment]
yes, I agree too. I will take a look at this in the next version.
